app/routes/lending/components/LendingAdmin.tsx

```diff
@@ -0,0 +1,132 @@
+import {
+  Button,
+  Card,
+  Flex,
+  Icon,
+  LoadingIndicator,
+} from '@webkom/lego-bricks';
+import { usePreparedEffect } from '@webkom/react-prepare';
+import { useState } from 'react';
+import { Helmet } from 'react-helmet-async';
+import { Link } from 'react-router-dom';
+import { fetchAllLendableObjects } from 'app/actions/LendableObjectActions';
+import { fetchAllLendingRequests } from 'app/actions/LendingRequestActions';
+import { Content } from 'app/components/Content';
+import NavigationTab from 'app/components/NavigationTab';
+import { selectAllLendableObjects } from 'app/reducers/lendableObjects';
+import { selectAllLendingRequests } from 'app/reducers/lendingRequests';
+import { useAppDispatch, useAppSelector } from 'app/store/hooks';
+import { LendingRequestStatus } from 'app/store/models/LendingRequest';
+import styles from './LendingAdmin.css';
+import { RequestItem } from './RequestItem';
+
+const LendableObjectsAdmin = () => {
+  const [showOldRequests, setShowOldRequests] = useState(false);
+
+  const dispatch = useAppDispatch();
+
+  usePreparedEffect(
+    'fetchObjects',
+    () => dispatch(fetchAllLendableObjects()),
+    [],
+  );
+
+  const lendableObjects = useAppSelector((state) =>
+    selectAllLendableObjects(state),
+  );
+  const fetchingObjects = useAppSelector(
+    (state) => state.lendableObjects.fetching,
+  );
+
+  usePreparedEffect(
+    'fetchRequests',
+    () => dispatch(fetchAllLendingRequests()),
+    [],
+  );
+
+  const lendingRequests = useAppSelector((state) =>
+    selectAllLendingRequests(state),
+  );
+
+  const fetchingRequests = useAppSelector(
+    (state) => state.lendingRequests.fetching,
+  );
+
+  return (
+    <Content>
+      <Helmet title="Utlån" />
+      <NavigationTab
+        title="Utlånsforespørsler"
+        back={{
+          label: 'Tilbake',
+          path: '/lending',
+        }}
+      />
+      <h2 className={styles.heading}>Ventende utlånsforespørsler</h2>
+      <LoadingIndicator loading={fetchingRequests}>
+        <Flex column gap={10} margin="0 0 30px">
+          {lendingRequests
+            .filter(
+              (request) => request.status === LendingRequestStatus.PENDING,
+            )
+            .map((request) => (
+              <RequestItem key={request.id} request={request} isAdmin />
+            ))}
+        </Flex>
+      </LoadingIndicator>
+
+      {showOldRequests ? (
+        <>
+          <h2 className={styles.heading}>Tidligere utlånsforespørsler</h2>
+          <Flex column gap={10} margin="0 0 30px">
+            {lendingRequests
+              .filter(
+                (request) => request.status !== LendingRequestStatus.PENDING,
+              )
```

Comment:
This filtering can't be right

Reply:
It is correct as the pending requests are shown above and then here we want to list others, aka. previously accepted or denied requests. However I agree this is not a very elegant solution.

Reply:
ah lol I didn't see the !==